### ISSUE TITLE
As a Developer, I want to see which step the Build fails at in Github

### ISSUE THREAD

**Issue by Nick9500** [Nick9500/ministocks#40]
As a Developer, I want to see which step the Build fails at in Github

[SP: 0.5]
[Priority: 1]
[Risk: High]

Demo
N/A (Invisible to User)

Tasks
Investigate how to show which step the build fails at in Circle/Travis.
Implement it.

**Comment by Nick9500** [comment]
As of InitTest commit, the build is passing and the text is as follows:
CircleCI - Your tests passed on CircleCI!
Travis - The Travis CI build passed

![capture](https://user-images.githubusercontent.com/4484946/37433703-4e65c9f4-27b3-11e8-9bdb-87c4584aec38.JPG)

**Comment by Nick9500** [comment]
As of CI - Purposely Fail Compilation commit, the build is failing and the text is as follows:
CircleCI - Your tests failed on CircleCI
Travis - The Travis CI build failed
![capture](https://user-images.githubusercontent.com/4484946/37433962-4f438b26-27b4-11e8-9dce-c9695d4d84a5.JPG)

**Comment by Nick9500** [comment]
As requested, I've distinguished between the build stages in CircleCI as can be seen below.
It has two stages (or workflows as CircleCI calls it) and when one fails, you'll be notified accordingly. I've set it so the BUILD step must pass in order for the TEST step to begin execution. Unfortunately, I cannot change the hardcoded text of "Your tests have passed!"/"Your tests failed" but the stage in which it failed will be clear with the red X or green check.

![afafa](https://user-images.githubusercontent.com/4484946/37437674-efe0b846-27c4-11e8-88a8-51ffb0b86af5.jpeg)
